src/config/user.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable no-console */
-
+// @ts-check
```

Comment:
Removing it would usually cause any typescript errors to be hidden in JS source, but there's a tsconfig.json in this repo that includes js files so it's not needed. Good catch.